### PR TITLE
Minor language fixes.

### PR DESCRIPTION
**Pull request by simonsticko**

### CODE REVIEW

File: 9.4/paper.tex [hunk 1]
```diff
@@ -563,7 +563,7 @@ \subsection{Advances in matrix-free infrastructure}\label{sec:mf}
 contexts. Examples are simulations with sharp interfaces (e.g., two-phase flow
 or shock capturing), where one needs to treat cells that are ``cut'' by
 the interface in a special way. A challenge is that cell batches
-might contain cut or non-cut cells, making vectorization of these operations potentially more complicated. Previous functionality has provided the option of masking certain cells in cell batches, which works well if
+might contain cut or non-cut cells, making the vectorization of these operations potentially more complicated. Previous functionality has provided the option of masking certain cells in cell batches, which works well if
```
Comment (bangerth) [inline 9.4/paper.tex:566]: Both of this sounds correct to me after 20 years speaking mostly English. But it doesn't hurt.

File: 9.4/paper.tex [hunk 2]
```diff
@@ -577,7 +577,7 @@ \subsection{Advances in matrix-free infrastructure}\label{sec:mf}
   taking two additional \texttt{std::function} arguments. The first function
   defines the operation to be scheduled on the vector entries before the
   matrix-vector product touches them, and the second what happens
-  afterwards. The new features also include a renumbering to maximize data
+  afterward. The new features also include a renumbering to maximize data
```
Comment (bangerth) [inline 9.4/paper.tex:580]: Interesting :-) https://grammarist.com/usage/afterward-afterwards/

**Review by bangerth — APPROVED** [review]

File: 9.4/paper.tex (hunk 2 above)
Comment (simonsticko) [reply 9.4/paper.tex:580]: I wouldn't have noticed this, but got it back when running the manuscript through [grammarly](grammarly.com). I assumed that American English was the preference.

Comment (bangerth) [reply 9.4/paper.tex:580]: Interesting. Yes, AE it is. Thanks for teaching all of us how to use a new tool!

**Pull request merged**